src/common.h

```diff
@@ -43,8 +43,7 @@ Nan::DefineOwnProperty(target, constant_name, constant_value, constant_attribute
 while (0)
 
 #define GLP_CREATE_HOOK_GUARDS(NAME) \
-    MemStatsGuard mguard{NAME->memstats_}; \
-    TermHookGuard hookguard{NAME->info_}
+    GLPKEnvStateGuard mguard{NAME->env_state_, NAME->info_}; \
```

Comment:
```suggestion
    GLPKEnvStateGuard mguard{(NAME)->env_state_, (NAME)->info_}; \
```

Reply:
I think the recommended practice is to put parens around any of the #define parameters. This prevents parsing issues if the parameter is actually an expression. For instance, I believe this would fail:

`GLP_CREATE_HOOK_GUARDS( (char *)ptr - MBD_SIZE )